app/src/main/res/values/strings.xml

```diff
@@ -89,12 +90,12 @@
     <string name="participant_flow_participant_id_hint">Scan Child ID QR Code</string>
     <string name="participant_flow_participant_id_scanned">The format of the ID is not supported</string>
     <string name="participant_flow_participant_id_description">Please scan the Child vaccination card with a QR-Code or manually enter the Child ID number:</string>
-    <string name="participant_flow_participant_id_label_manual_entry">Manual ID Entry</string>
-    <string name="participant_flow_participant_id_input_hint">#1234567890</string>
+    <string name="participant_flow_participant_id_label_manual_entry">Manual ID Entry or Search by Name of the Mother </string>
```

Comment:
Hello @druchniewicz  , we need to add a query that searches name of the mother here. You can guide me on this. this is something that came up today.